### ISSUE TITLE
Missing/bad value (NAN) should not be included in scaling - special case

### ISSUE THREAD

**Issue by DG12** [ruuvi/com.ruuvi.station.webui#149]
Sensor BB12 was not returning valid humidity values.
After resetting (hard way, remove battery) now returning valid data.

Graph for any time period which  includes any NAN points does not show graph line.
Sliding cursor the after reset shows correct humidity within the 45% range but no graph is shown.
Selecting last 20 minutes which only contains valid data and graph is presented nicely
<img width="1512" alt="no_graph_for_BB12" src="https://user-images.githubusercontent.com/2365961/155921465-77fc4682-46d2-40e9-8bf7-3de22ae14e4e.png">
<img width="756" alt="graph_for_BB12" src="https://user-images.githubusercontent.com/2365961/155921474-0eb3f173-a10c-4787-a7a7-aa436a205df4.png">
.

**Comment by laurijamsa** [comment]
If I understand this issue correctly, we are not showing a graph at all if any of the values that should be visible in the graph is NA. Let's try to adjust this so that the graph would be shown, even if some of the data points would be NA. Those NA values just should be treated as no data points available.

**Comment by io53** [comment]
The invalid datapoints are now filtered out, but its a bit hard to test if you don't have such data

**Comment by laurijamsa** [comment]
I didn't test as I don't have such data. But let's deploy anyways as you have tested this already to work.

**Comment by markoaamunkajo** [comment]
I also don't have such sensor data now.

**Comment by DG12** [comment]
I tried to look further into the problem but at this time the graph is very different.
The time period around the problem area (see original graph posted above) 
now shows no data points rather a straight line.
i.e. between 25 Feb ( 25.2 ) and 2022-02-27 22:30
<img width="1280" alt="BB12badHumGraph" src="https://user-images.githubusercontent.com/2365961/165181282-c57de654-53be-43bd-a6b1-3c3156c5be10.png">

@Lauri I have shared this sensor with you .
@markoaamunkajo I can share it with you if you give me your email address.
